### DISCUSSION TITLE
Cannot import Splink

### DISCUSSION THREAD

**Discussion by lenroc14** [moj-analytical-services/splink#640] [General]
Hello, I have been using splink for several months now. I have done the following many times with no issues:

`pip install splink`

followed by 

`from splink import Splink`

However, for some reason this morning I am receiving the error:

 cannot import name 'Splink' from 'splink' (/local_disk0/.ephemeral_nfs/envs/pythonEnv-92cd1ad2-cb6f-4c9b-a13e-19b329d4025d/lib/python3.7/site-packages/splink/__init__.py)

I am wondering how to resolve this. Thank you!

**Comment by RobinL** [comment]
It's because splink 3 has just been released which is not backwards compatible.  You need to either install a 2.x version, or refer to the new docs for how to use splink 3 https://moj-analytical-services.github.io/splink/

**Reply by lenroc14** [reply]
I see. Thank you!
